neuromaps/datasets/data/osf.json

```diff
@@ -2087,6 +2106,42 @@
                 "60b92267cb2a5e032768e864"
             ]
         },
+        {
+            "source": "sydnor2021",
+            "desc": "SAaxis",
+            "space": "fsLR",
+            "den": "32k",
+            "hemi": "L",
+            "format": "surface",
+            "fname": "source-sydnor2021_desc-SAaxis_space-fsLR_den-32k_hemi-L_feature.func.gii",
+            "rel_path": "sydnor2021/SAaxis/fsLR/",
+            "checksum": "b15cca875e26a4d5281e4c0f7de698e7",
+            "title": null,
+            "tags": null,
+            "redir": null,
+            "url": [
+                "4mw3a",
+                "61e06d4ac99ebd0254018176"
+            ]
+        },
+        {
+            "source": "sydnor2021",
+            "desc": "SAaxis",
+            "space": "fsLR",
+            "den": "32k",
+            "hemi": "R",
+            "format": "surface",
+            "fname": "source-sydnor2021_desc-SAaxis_space-fsLR_den-32k_hemi-L_feature.func.gii",
```

Comment:
Should this filename be "...hemi-R_feature", instead of "...hemi-L_feature"?

Reply:
yES thx fixing now 😅